__tests__/Unit/Components/standup/standup.test.tsx

```diff
@@ -0,0 +1,59 @@
+import StandUp from '@/pages/standup';
+import { RouterContext } from 'next/dist/shared/lib/router-context';
```

Comment:
do we need routercontext for mocking?

Reply:
I thought that the renderWithRouter function by @bhtibrewal  already did that

Reply:
This is fixed